### PR TITLE
Make many components resolution-independent

### PR DESCRIPTION
**Pull request by grapegravity**
- implements a `pixelsToEm` Sass function, which converts pixel values to `em` based on our default font size of 16px
- uses said function to change all measurements in affected components to `em`s so they can be scaled to any size
- improves inheritance of font sizes in some cases
- adds a `forceMobile` property to the `Modal`, which allows you to force the mobile viewport style
  - this was added to work around the fact that in Bugherd's scaling model, the media queries don't scale with the page. More work might allow us to fix this in CSS, but at this point the JS solution seems best

### CODE REVIEW

File: packages/modal/index.jsx [hunk 1]
```diff
@@ -81,6 +81,7 @@ module.exports = React.createClass({
     modalClasses.addModifier({
       'visible': this.state.showModal,
       'invisible': !this.state.showModal,
+      'forceMobile': this.props.forceMobile === true,
```
Comment (mmilo) [inline packages/modal/index.jsx]: @grapegravity is this related to resolution stuff?

Comment (grapegravity) [inline packages/modal/index.jsx]: Yep, this forces it to use the styles which are usually used in mobile breakpoints.

As noted in the PR description, this it to work around the fact that media queries _don’t_ scale!

Comment (mmilo) [inline packages/modal/index.jsx]: kk

**Comment by mmilo** [comment]
:+1:
